.github/workflows/codeql-analysis.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       matrix:
         # Override automatic language detection by changing the below list
         # Supported options are ['csharp', 'cpp', 'go', 'java', 'javascript', 'python']
-        language: ['cpp']
+        language: ['cpp', 'csharp', 'java', 'javascript', 'python']
```

Comment:
Does CodeQL not support objc?

Reply:
Seems objc is not in the list here.

https://codeql.github.com/docs/codeql-language-guides/